### PR TITLE
community/fwsnort: update to 1.6.7 / add check()

### PR DESCRIPTION
**Pull request by itoffshore**
community/fwsnort: update to `1.6.7`

### CODE REVIEW

File: community/fwsnort/APKBUILD [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-pkgver=1.6.6
+pkgver=1.6.7
 pkgrel=0
 pkgdesc="Application layer IDS/IPS by translating snort rules into iptables"
 url="http://www.cipherdyne.org/fwsnort/"
 arch="noarch"
 license="GPL"
 depends="perl perl-netaddr-ip perl-iptables-parse iptables ip6tables net-tools wget"
 install="$pkgname.post-install"
+options="!check"
```
Comment (Shizmob) [inline community/fwsnort/APKBUILD]: Does [test/test-fwsnort.pl](https://github.com/mrash/fwsnort/blob/master/test/test-fwsnort.pl) not work?

**Comment by itoffshore** [comment]
`check()` added

**Comment by Shizmob** [comment]
Hmm, apparently there's some test failures there. Can/did you verify that the package works? It would be ideal if we could get the tests to pass as well. :)

**Comment by itoffshore** [comment]
I got a few more tests to pass running them as `root` :

```
musl64 [~/aports/community/fwsnort/src/fwsnort-nodeps-1.6.7/test]$ sudo ./test-fwsnort.pl

[+] Starting the fwsnort test suite...

    args: 

    Saved results from previous run to: output.last/

[install] test directory: fwsnort-install...........................pass (1)
[compilation] fwsnort compiles......................................pass (2)
[operations] --help.................................................pass (3)
[operations] --dump-conf............................................pass (4)
[operations] --ipt-list.............................................pass (5)
[operations] --ipt-check-capabilities...............................pass (6)
[operations] --snort-sid 108 EXTERNAL->HOME.........................fail (7)
[operations] --snort-sid 1292 HOME->EXTERNAL........................fail (8)
[operations] multiple rules --snort-sid 108,109,321.................fail (9)
[operations] --snort-sid badsid.....................................pass (10)
[operations] --include-type backdoor................................fail (11)
[operations] --strict --include-type backdoor.......................fail (12)
[operations] --include-type emerging-all............................fail (13)
[operations] multiple files --include-type backdoor,dns,ftp.........fail (14)
[operations] --exclude-type emerging-all............................fail (15)
[operations] multiple --exclude-type emerging-all,backdoor,dns,ftp..fail (16)
[operations] --include-type backdoor,dns,ftp --exclude-type dns.....fail (17)
[operations] --snort-sid 108,109,321 --exclude-regex sid:109........fail (18)
[operations] --snort-sid 108,109,321 --include-regex sid:109........fail (19)
[operations] ip6tables --snort-sid 108..............................fail (20)
[operations] ip6tables  --snort-sid 108,109,321.....................fail (21)
[operations] ip6tables --snort-sid badsid...........................pass (22)
[operations] ip6tables --include-type backdoor......................fail (23)
[operations] ip6tables --strict --include-type backdoor.............fail (24)
[operations] ip6tables --include-type emerging-all..................fail (25)
[operations] ip6tables --include-type backdoor,dns,ftp..............fail (26)
[operations] ip6tables --exclude-type emerging-all..................fail (27)
[operations] ip6tables --ex... emerging-all,backdoor,dns,ftp........fail (28)
[operations] ip6tables --in.. backdoor,dns,ftp --ex.. dns...........fail (29)
[operations] ip6tables --sn.. 108,109,321 --ex.. sid:109............fail (30)
[operations] ip6tables --sn.. 108,109,321 --in.. sid:109............fail (31)
[errors] look for perl warnings.....................................pass (32)

[+] passed/failed/executed: 9/23/32 tests

musl64 [~/aports/community/fwsnort/src/fwsnort-nodeps-1.6.7/test]$ ./test-fwsnort.pl
[*] ./test-fwsnort.pl: You must be root (or equivalent UID 0 account) to effectively test fwsnort at ./test-fwsnort.pl line 822.
```

**Comment by itoffshore** [comment]
it works ok:

```
musl64 [~/aports/community/fwsnort]$ sudo apk add ~/packages/comm*/x*/fwsnort-1.6.7-r0.apk
(1/10) Installing perl-netaddr-ip (4.079-r0)
(2/10) Installing perl-iptables-parse (1.6.1-r0)
(3/10) Installing libmnl (1.0.4-r0)
(4/10) Installing libnftnl-libs (1.0.7-r0)
(5/10) Installing iptables (1.6.1-r0)
(6/10) Installing ip6tables (1.6.1-r0)
(7/10) Installing mii-tool (1.60_git20140218-r1)
(8/10) Installing net-tools (1.60_git20140218-r1)
(9/10) Installing wget (1.19.1-r2)
(10/10) Installing fwsnort (1.6.7-r0)
Executing fwsnort-1.6.7-r0.post-install

FWSNORT installed & daily signature updates configured.

To update now run: 	/usr/bin/update-fwsnort
 
Executing busybox-1.26.2-r3.trigger
OK: 417 MiB in 153 packages
musl64 [~/aports/community/fwsnort]$ sudo update-fwsnort
[+] Downloading latest rules into /etc/fwsnort/snort_rules/--2017-05-13 03:50:00--  http://rules.emergingthreats.net/open/snort-2.9.0/emerging-all.rules
Resolving rules.emergingthreats.net... 204.12.217.19, 96.43.137.99
Connecting to rules.emergingthreats.net|204.12.217.19|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 9892964 (9.4M)
Saving to: 'emerging-all.rules'

emerging-all.rules                      100%[===============================================================================>]   9.43M  5.66MB/s    in 1.7s    

2017-05-13 03:50:02 (5.66 MB/s) - 'emerging-all.rules' saved [9892964/9892964]

[+] Finished.
[+] Testing /sbin/iptables for supported capabilities...
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
    Snort Rules File          Success   Fail      Total     

[+] emerging-all.rules        10955     6442      17397     
                              =============================
                              10955     6442      17397     

[+] Generated iptables rules for 10955 out of 17397 signatures: 62.97%

[+] Logfile: /var/log/fwsnort/fwsnort.log
[+] iptables script (individual commands): /var/lib/fwsnort/fwsnort_iptcmds.sh


    Main fwsnort iptables-save file: /var/lib/fwsnort/fwsnort.save

    You can instantiate the fwsnort policy with the following command:

    /sbin/iptables-restore < /var/lib/fwsnort/fwsnort.save

    Or just execute: /var/lib/fwsnort/fwsnort.sh

 
[+] Splicing fwsnort 10955 rules into the iptables policy...
musl64 [~/aports/community/fwsnort]$ 

```

**Pull request closed without merging**